packages/server/src/stages/rewrite-imports/index.ts

```diff
@@ -0,0 +1,55 @@
+import {Stage, transform} from "@blitzjs/file-pipeline"
+import path from "path"
+
+const isJavaScriptFile = (filepath: string) => filepath.match(/\.(ts|tsx|js|jsx)$/)
+
+const isInAppFolder = (s: string, cwd: string) => s.replace(cwd + path.sep, "").indexOf("app") === 0
+
+/**
+ * Returns a Stage that converts relative files paths to absolute
+ */
+export const createStageRewriteImports: Stage = ({config: {cwd}}) => {
```

Comment:
This stage is very similar to https://github.com/blitz-js/blitz/blob/07b7a8a14809097d119c7b080a31abea10ca3d58/packages/server/src/stages/relative/index.ts, maybe we can refactor them to share code.